docs/sql/statements/create-table.rst

```diff
@@ -683,17 +683,6 @@ Sets size of transaction log prior to flushing.
   Size (bytes) of translog.
 
 
-.. _sql-create-table-translog-interval:
```

Comment:
Why do we remove this? This setting is relevant, if the translog durability is set to async

Reply:
sorry, mistook this for the translog sync invertal